services/graphite/service.go

```diff
@@ -34,6 +35,7 @@ const (
 	statBatchesTransmitFail = "batchesTxFail"
 	statConnectionsActive   = "connsActive"
 	statConnectionsHandled  = "connsHandled"
+	statPointsNaNFail       = "pointsNaNFail"
```

Comment:
Done. I wasn't sure if you guys preferred grouping based on relationship or just append only for lists of constants.